package.json

```diff
@@ -14,8 +14,12 @@
     "@babel/plugin-transform-shorthand-properties": "^7.0.0",
     "@babel/plugin-transform-spread": "^7.0.0",
     "@babel/plugin-transform-template-literals": "^7.0.0",
+    "@babel/preset-env": "^7.7.5",
+    "@babel/preset-react": "^7.7.4",
+    "@babel/preset-typescript": "^7.7.4",
     "@typescript-eslint/eslint-plugin": "^2.5.0",
     "@typescript-eslint/parser": "^2.5.0",
+    "babel-jest": "^24.9.0",
```

Comment:
Is it necessary to add all these babel modules? If we're running the Jest tests on a recent version of Node, all of the syntax should be valid, right?

Reply:
I've removed babel-jest etc for now. They will need adding back later when we introduce TypeScript.